test/Makefile

```diff
@@ -0,0 +1,36 @@
+FILE = function-names
+MINFILE = $(FILE).min
+
+# wasm function name and arguments
+FUNCTION = _Z4add1i
+ARGS = 10
+
+# Compile `test.rs` and execute in Node.
+all:
+	make gc
+	make eval
+	make wat
+	make print
+
+clean:
+	rm *.wat
```

Comment:
Add `-f` flag so it doesn't error if there are no `.wat` files.

Reply:
Thanks!